<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>850</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="TEXT">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>10</y>
     <width>300</width>
     <height>160</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>판 크기 설정</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>780</y>
     <width>100</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>뒤로 가기</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="select_pan">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>240</y>
     <width>441</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="selectbutton">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>340</y>
     <width>100</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>결정</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
